src/scripts/main.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+
+  if (!wall.contains(e.target)) {
+    return;
+  }
+
+  let x = e.offsetX - spider.offsetWidth / 2;
+  let y = e.offsetY - spider.offsetHeight / 2;
+
+  const CoordsX = wall.clientWidth - spider.offsetWidth;
+  const CoordsY = wall.clientHeight - spider.offsetHeight;
```

Comment:
classes begin with a capital letter, but not simple variables
```suggestion
  const coordsX = wall.clientWidth - spider.offsetWidth;
  const coordsY = wall.clientHeight - spider.offsetHeight;
```